src/components/menus/SmartAutoCorrect.ts

```
@@ -0,0 +1,804 @@
+        
+        const button = this.shadowRoot.querySelector('.smart-auto-correct');
+        if (!button) return;
+        
+        if (!this.editor) return;
+        
+        const extensions = this.editor.extensionManager.extensions;
+        const smartAutoCorrectExt = extensions.find((ext: AnyExtension): ext is SmartAutoCorrectExtension => 
+            ext.name === 'smartAutoCorrect' && 'options' in ext
+        );
+        
+        if (smartAutoCorrectExt && smartAutoCorrectExt.options.enabled) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)